src/appkit.cs

```diff
@@ -7216,10 +7216,22 @@ interface NSGridView
 		[Static]
 		[Export ("gridViewWithNumberOfColumns:rows:")]
 		NSGridView Create (nint columnCount, nint rowCount);
-
+		
+#if !XAMCORE_4_0
+		[EditorBrowsable (EditorBrowsableState.Never)]
+		[Obsolete]
```

Comment:
The Obsolete attribute needs a description (to tell the user which overload to use instead).